NetKAN/SDHI-ServiceModuleSystem.netkan

```diff
@@ -1,11 +1,11 @@
 {
-    "spec_version" : 1,
+    "spec_version" : "v1.4",
     "identifier"   : "SDHI-ServiceModuleSystem",
     "$kref"        : "#/ckan/github/sumghai/SDHI_ServiceModuleSystem",
     "name"         : "Sum Dum Heavy Industries - Service Module System",
     "abstract"     : "Service Module and accessories designed specifically for use with the stock Mk1-2 Command Pod, vaguely resembling NASA's Orion MPCV. May or may not have a convincing stockalike appearance.",
     "license"      : "CC-BY-SA-4.0",
-    "ksp_version"  : "0.90",
+    "ksp_version"  : "1.0",
```

Comment:
`1.0` will match `1.0.<ANYTHING>`, including future versions. Is that what you want or?:
```json
{
    "ksp_version_min": "1.0.0",
    "ksp_version_max": "1.0.2"
}
```